language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/InvocationNodeContextProvider.java

```diff
@@ -110,18 +111,40 @@ public void sort(BallerinaCompletionContext context, T node, List<LSCompletionIt
         }
 
         List<String> existingNamedArgs = CommonUtil.getDefinedArgumentNames(context, params.get(), argumentNodeList);
-        Predicate<ParameterSymbol> predicate = parameter -> parameter.paramKind() == ParameterKind.REQUIRED
-                || parameter.paramKind() == ParameterKind.DEFAULTABLE;
-        params.get().stream().filter(predicate).forEach(parameterSymbol -> {
-            Optional<String> paramName = parameterSymbol.getName();
-            TypeSymbol paramType = parameterSymbol.typeDescriptor();
-            String defaultValue = CommonUtil.getDefaultValueForType(paramType).orElse("");
-            if (paramName.isEmpty() || paramName.get().isEmpty() || existingNamedArgs.contains(paramName.get())) {
-                return;
+        for (ParameterSymbol parameterSymbol : params.get()) {
+            if (parameterSymbol.paramKind() == ParameterKind.REQUIRED ||
+                    parameterSymbol.paramKind() == ParameterKind.DEFAULTABLE) {
+                Optional<String> paramName = parameterSymbol.getName();
+                TypeSymbol paramType = parameterSymbol.typeDescriptor();
+                String defaultValue = CommonUtil.getDefaultValueForType(paramType).orElse("");
```

Comment:
Since we don't have any other usages for `paramType`, can we use it straightaway in the default value generation without assigning to a variable?

Reply:
Done